pkg/controllers/clusterexternalsecret/clusterexternalsecret_controller.go

```diff
@@ -215,6 +215,15 @@ func (r *Reconciler) createOrUpdateExternalSecret(ctx context.Context, clusterEx
 	mutateFunc := func() error {
 		externalSecret.Labels = esMetadata.Labels
 		externalSecret.Annotations = esMetadata.Annotations
+		if value, ok := clusterExternalSecret.Annotations["force-sync"]; ok {
```

Comment:
nit: let's make this annotation a `const` , and in the format : `external-secrets.io/force-sync`. We can update docs to reflect that as well, and it will be more clear to any user seeing this is the annotation that triggered it.

Reply:
IMHO this shouldn't be a nit. The force sync annotation is something we would be stuck with for quite a while and it should be documented before merge, including its behavior

Reply:
Gladly, that matches what I had initially: https://github.com/external-secrets/external-secrets/commit/b0ec13b9f31707d33ba5806a177e58f17060c883#diff-48b6ea2bfdb10c1e1c6d5a543fde5d5d0a9c2208bcb4e3d37ea75730ed6517c1